### DISCUSSION TITLE
Uploading images via the online editor: Rederict user-images.githubusercontent to repo

### DISCUSSION THREAD

**Discussion by ratnanil** [community/community#23840] [New to GitHub]
<p>I have a lot of projects hosted on github where I’m working with non git users. Much of the contributions by these users can be made by using the github online editor, which is great. This online editor even supports the GUI based insertion of images by uploading these to <a href="http://user-images.githubusercontent.com/user-id/some-hash.png" rel="noopener nofollow ugc">user-images.githubusercontent.com/user-id/some-hash.png</a> and inserting them with markdown syntax.</p>
<p>Is there a way to redirect the automatic upload to the github repo, rather than the users’s own storage? Maybe by tweaking something in the <code>.github</code> folder? This would greatly enhance the collaboration between git and non git users on the same project, since all images associated with the project would in fact be stored in the specific repo.</p>

**Comment by airtower-luna** [comment]
<p>How are the users uploading the images? It should be possible to include them directly in a pull request.</p>

**Reply by Ningaraj36** [reply]
Hi there Ningaraj here
self thaught java back end developer

**Comment by ratnanil** [comment]
<p>They upload the images by dragging and dropping them into the github online editor. Or by pasting the image from  their clipboard into the github online editor… do you know how I mean?</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="ratnanil" data-post="3" data-topic="203835">
<div class="title">
<div class="quote-controls"></div>
 ratnanil:</div>
<blockquote>
<p>github online editor</p>
</blockquote>
</aside>
<p>I’m not sure what you mean here. Do you mean the one you get when you edit a file in the browser?</p>

**Comment by ratnanil** [comment]
<p>I’ve made a quick gif to illustrate what I mean:</p>
<p></p><div class="large-image-placeholder"><a href="https://user-images.githubusercontent.com/12532091/135542114-0dc6044b-43d2-455e-b7fe-85b838b69a2b.gif" target="_blank" rel="noopener nofollow ugc"><svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="url">https://user-images.githubusercontent.com/12532091/135542114-0dc6044b-43d2-455e-b7fe-85b838b69a2b.gif</span><span class="help">(image larger than 4 MB)</span></a></div>
<p>The image was uploaded to <a href="https://user-images.githubusercontent.com/12532091/135541406-7141a8d7-7bf9-4c82-a4d9-ea4b7fe1fb34.jpg" rel="noopener nofollow ugc">https://user-images.githubusercontent.com/12532091/135541406-7141a8d7-7bf9-4c82-a4d9-ea4b7fe1fb34.jpg</a><br>
Instead of the users ID (or whatever <code>12532091</code> is), I would like to upload the image into the repo that holds the file into which the image was just copied.</p>
<p>Is my question clearer now?</p>

**Reply by MasterInQuestion** [reply]
Such design may trap this:
    https://github.com/orgs/community/discussions/54551#discussioncomment-9284501

**Comment by airtower-luna** [comment]
<p>It is. You can use “Upload files” for that:<br>
<img src="https://user-images.githubusercontent.com/5814919/181079658-95f05136-e979-4336-8576-5f78002f0c26.png" alt="grafik" data-base62-sha1="py7iltwNpf1DIQ2z1Znmtf8pry8" width="352" height="118"></p>

**Comment by ratnanil** [comment, marked as answer]
<p>I was specifically looking for a way to do it with copy and paste (or drag and drop) since my non git collaborators are ms word users and not too keen on the exra steps involved. But thanks for your answer!</p>

**Reply by dustinfreeman** [reply]
Hey from 2024! Did you ever find another solution for this?
I'm looking for the exact same solution you are; I want to redirect images dragged into an .md file in the GitHub UI to a location in the repo, as opposed to https://github.com/user-attachments/...
I'm also working on some novel photo management solutions

**Comment by Grumpy2869** [comment]
All I can say is that I am sorry for the mix up, I am still really new to this. And pretty much self taught. I learn from my mistakes and grow. 
I Apologize sincerely I am sorry

**Comment by jumpjack** [comment]
How can I see the list of images I uploaded/pasted to user-images.githubusercontent.com? I see all have always same id after ".githubusercontent.com", so I guess they are all associated to me (I pasted them while logged into my account), but how can I list and manage them?

**Reply by SaswataGA** [reply]
Maybe GitHub doesn’t let you manage or list images uploaded to `user-images.githubusercontent.com`. so to find them, maybe you’ll need to check your past issues, pull requests, or your browser history.

**Reply by MasterInQuestion** [reply]
A repository specifically for attachments.
    See also: https://github.com/orgs/community/discussions/41225#discussioncomment-10713872

**Comment by Varun13245** [comment]
![Uploading 20240327_224906.jpg…]()

**Comment by DanteTerry** [comment]
![bitway stats](https://github.com/community/community/assets/112513589/e6113d99-205b-4c9a-bdc4-30bd306b60c2)

**Comment by wanwanwak** [comment]
![sosowmaowamow](https://github.com/user-attachments/assets/f1f34fab-1317-432f-96bc-66eb7b5d30aa)

**Comment by wanwanwak** [comment]
![Uploading wmawlamwoawma.png…]()

**Comment by wanwanwak** [comment]
<img width="537" alt="wmawlamwoawma" src="https://github.com/user-attachments/assets/ff9a4b55-6305-4807-905c-9eaffb367200">

**Comment by wanwanwak** [comment]
![titamwamowa](https://github.com/user-attachments/assets/ed13590e-8062-4433-95e9-7fb59674d4bc)

**Comment by wanwanwak** [comment]
![Uploading nasmawamowamo.jpg…]()

**Comment by wanwanwak** [comment]
![nasmawamowamo](https://github.com/user-attachments/assets/8c1d19ac-a7d0-486d-bf9f-1be9d57a26c8)

**Comment by wanwanwak** [comment]
<img width="537" alt="mlwamlwam" src="https://github.com/user-attachments/assets/ecf5d511-9152-4709-9cd3-84684effe6d6">

**Comment by wanwanwak** [comment]
<img width="537" alt="maya" src="https://github.com/user-attachments/assets/145a2774-8b5a-41dc-bc45-22351be46fae">

**Reply by wanwanwak** [reply]
[kamzkmlzakml](https://google.com)

**Comment by wanwanwak** [comment]
<img width="537" alt="tasalolmwam" src="https://github.com/user-attachments/assets/70eea086-5b7f-45fb-956a-7ea1756a83d8">

**Reply by wanwanwak** [reply]
<img width="537" alt="tasalolmwam" src="https://github.com/user-attachments/assets/30fa6a91-5f7e-443b-9939-26180081e3b3">

**Comment by wanwanwak** [comment]
![wmalmwal](https://github.com/user-attachments/assets/233931a5-a0a4-4c81-8546-be645f7e5974)

**Comment by wanwanwak** [comment]
<img width="500" alt="mwabwalwam" src="https://github.com/user-attachments/assets/20ae61df-ceeb-45b2-a2f5-33755d529af0">

**Comment by MissAri1** [comment]
![Uploading 48346581-A23C-4489-86D0-AAE284006C28.jpeg…]()

**Reply by MissAri1** [reply]
https//:githubusercontent.com.jpeg

**Comment by wanwanwak** [comment]
<img width="587" alt="abooora" src="https://github.com/user-attachments/assets/cc5bd858-a2be-4710-8357-4bbee291c3b4" />

**Reply by wanwanwak** [reply]
https://bing.com[](url)
